rules-tests/Php74/Rector/Closure/ClosureToArrowFunctionRector/Fixture/keep_docblock_on_return.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function() {
+    /** @psalm-suppress UndefinedFunction */
+    return ff();
+};
+?>
+-----
+<?php
+
+fn() => /** @psalm-suppress UndefinedFunction */
```

Comment:
keep start comment on current line is on purpose to avoid make creating a tab spaces which will fail on compare in test.